packages/vuetify/src/components/VChip/VChip.sass

```diff
@@ -106,24 +110,16 @@
   .v-icon
     color: inherit
 
+.v-chip--link
   &:before
```

Comment:
```suggestion
  &::before
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/::before#Syntax

Reply:
This will need to be a more global change considering 90% of the framework uses &:before/&:after